app/stores/basicPokemonDetail.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+let fs = require('fs'),
+    basicPokemonDetail = require(__appbase + '/models/BasicPokemonDetail.js'),
+    pokemonListPath = require(__resourcebase + '/pokemonlist.json');
+
+module.exports = {
+
+    fill: function loadBasicPokemonDetails() {
+        logger.info('Loading Basic Pokemon Details');
+
+        let data = JSON.stringify(pokemonListPath, function (key, value) {
+            var pokemonID, pokemonName, gender, male_ratio, female_ratio, breedable;
+            if (key != 'name' && typeof value.name !== "undefined") {
+                pokemonID = key;
+                pokemonName = value.name;
+                gender = value.gender.abbreviation;
+                male_ratio = value.gender.male_ratio;
+                female_ratio = value.gender.female_ratio;
+                breedable = value.gender.breedable;
+
+
+                basicPokemonDetail.create({
```

Comment:
Only db operations were needed to be moved to the stored other part could be still there in the filler/basicPokemondetails.js.  Its a suggestion since we usually keep database operations from the main file in thie being the filler//basicPokemondetails.js